### ISSUE TITLE
Support for Mutual exclusion

### ISSUE THREAD

**Issue by ignatenkobrain** [kislyuk/argcomplete#121]
https://docs.python.org/3.5/library/argparse.html#mutual-exclusion
```python
parser = argparse.ArgumentParser(prog='PROG')
group = parser.add_mutually_exclusive_group()
group.add_argument('--foo', action='store_true')
group.add_argument('--bar', action='store_false')
```
If we already have autocompleted `--foo` we should not suggest `--bar`

**Comment by evanunderscore** [comment]
#144 was a duplicate of this - this was fixed in #145

**Comment by kislyuk** [comment]
Thanks, released in v1.5.1, closing.
